<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>843</width>
    <height>472</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QScrollArea" name="scrollArea">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>30</y>
      <width>501</width>
      <height>311</height>
     </rect>
    </property>
    <property name="widgetResizable">
     <bool>true</bool>
    </property>
    <widget class="QWidget" name="scrollAreaWidgetContents">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>499</width>
       <height>309</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item alignment="Qt::AlignHCenter">
       <widget class="QLabel" name="labPic">
        <property name="text">
         <string>TextLabel</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>843</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonTextUnderIcon</enum>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actAddDir"/>
   <addaction name="actAddFile"/>
   <addaction name="actDelNode"/>
   <addaction name="actScanNode"/>
   <addaction name="separator"/>
   <addaction name="actZoomIn"/>
   <addaction name="actZoomOut"/>
   <addaction name="actFitReal"/>
   <addaction name="actFitW"/>
   <addaction name="actFitH"/>
   <addaction name="separator"/>
   <addaction name="actFloat"/>
   <addaction name="actVisiable"/>
   <addaction name="actQuit"/>
  </widget>
  <widget class="QDockWidget" name="dockWidget">
   <property name="windowTitle">
    <string>图片目录树</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>1</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QTreeWidget" name="treeFile">
       <column>
        <property name="text">
         <string>节点</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>节点类型</string>
        </property>
       </column>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <action name="actAddDir">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/images/icons/icons/open3.bmp</normaloff>:/images/icons/icons/open3.bmp</iconset>
   </property>
   <property name="text">
    <string>添加目录</string>
   </property>
   <property name="menuRole">
    <enum>QAction::NoRole</enum>
   </property>
  </action>
  <action name="actAddFile">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/images/icons/icons/824.bmp</normaloff>:/images/icons/icons/824.bmp</iconset>
   </property>
   <property name="text">
    <string>添加文件</string>
   </property>
   <property name="menuRole">
    <enum>QAction::NoRole</enum>
   </property>
  </action>
  <action name="actDelNode">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/images/icons/icons/delete1.bmp</normaloff>:/images/icons/icons/delete1.bmp</iconset>
   </property>
   <property name="text">
    <string>删除节点</string>
   </property>
   <property name="menuRole">
    <enum>QAction::NoRole</enum>
   </property>
  </action>
  <action name="actScanNode">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/images/icons/icons/fold.bmp</normaloff>:/images/icons/icons/fold.bmp</iconset>
   </property>
   <property name="text">
    <string>遍历节点</string>
   </property>
   <property name="menuRole">
    <enum>QAction::NoRole</enum>
   </property>
  </action>
  <action name="actZoomIn">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/images/icons/icons/418.bmp</normaloff>:/images/icons/icons/418.bmp</iconset>
   </property>
   <property name="text">
    <string>放大</string>
   </property>
   <property name="menuRole">
    <enum>QAction::NoRole</enum>
   </property>
  </action>
  <action name="actZoomOut">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/images/icons/icons/416.bmp</normaloff>:/images/icons/icons/416.bmp</iconset>
   </property>
   <property name="text">
    <string>缩小</string>
   </property>
   <property name="menuRole">
    <enum>QAction::NoRole</enum>
   </property>
  </action>
  <action name="actFitReal">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/images/icons/icons/414.bmp</normaloff>:/images/icons/icons/414.bmp</iconset>
   </property>
   <property name="text">
    <string>实际大小</string>
   </property>
   <property name="menuRole">
    <enum>QAction::NoRole</enum>
   </property>
  </action>
  <action name="actFitW">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/images/icons/icons/426.bmp</normaloff>:/images/icons/icons/426.bmp</iconset>
   </property>
   <property name="text">
    <string>适合宽度</string>
   </property>
   <property name="menuRole">
    <enum>QAction::NoRole</enum>
   </property>
  </action>
  <action name="actFitH">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/images/icons/icons/424.bmp</normaloff>:/images/icons/icons/424.bmp</iconset>
   </property>
   <property name="text">
    <string>适合高度</string>
   </property>
   <property name="menuRole">
    <enum>QAction::NoRole</enum>
   </property>
  </action>
  <action name="actFloat">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/images/icons/icons/814.bmp</normaloff>:/images/icons/icons/814.bmp</iconset>
   </property>
   <property name="text">
    <string>窗口浮动</string>
   </property>
   <property name="menuRole">
    <enum>QAction::NoRole</enum>
   </property>
  </action>
  <action name="actVisiable">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/images/icons/icons/BEBULB.ICO</normaloff>:/images/icons/icons/BEBULB.ICO</iconset>
   </property>
   <property name="text">
    <string>窗口可见</string>
   </property>
   <property name="menuRole">
    <enum>QAction::NoRole</enum>
   </property>
  </action>
  <action name="actQuit">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/images/icons/icons/exit.bmp</normaloff>:/images/icons/icons/exit.bmp</iconset>
   </property>
   <property name="text">
    <string>退出</string>
   </property>
   <property name="menuRole">
    <enum>QAction::NoRole</enum>
   </property>
  </action>
 </widget>
 <resources>
  <include location="res.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>actQuit</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>421</x>
     <y>235</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
